tests/frontend/test_kibana_app/cypress/integration/utils/driver.js

```diff
@@ -27,6 +27,11 @@ export const fillField = (selector, text) => {
   return this;
 };
 
+export const fillSelectorField = (selector, text) => {
```

Comment:
this function performs almost the same function as fillField (which also clears the field before inputting anything.
should we rename that something like ClearAndFillField, or delete this one and just use that one?

Reply:
I had to create this function because I had a problem to perform an input on the filter fields.
The reason was related to a nonclearable element.
We can analyze the particular issue and took the best decision with the team.

Reply:
I think that we can rename the previous function to ClearandFillField